publish_job.py

```diff
@@ -62,21 +55,46 @@ def get_sts_and_sns_clients(aws_auth_method):
     return sts_client, client
 
 
+def find_output():
+    """
+    This function parses the stdout file and extracts the workflow output
+    :return:
+    """
+    # open stdout file
+    f = open("stdout.txt", "r")
```

Comment:
Fixed the file name. Should have been `_stdout.txt`. I agree. Currently the way everything is written into stdout is not ideal for us to figure out which workflow steps have run (because there is no indication in the content). Also since the actual CWL file is not in the work directory it's been a challenge to know what the output field is called.